source_actblue_csv_api/source.py

```diff
@@ -93,6 +93,20 @@ def path(
     
     def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
         return None
+    
+    def backoff_time(self, response: requests.Response) -> Optional[float]:
+        """
+        The ActBlue CSV API has a rate limit of 10 requests per minute. Since we
+        have to split requests into >6 month intervals this can end up hitting that 
```

Comment:
I think this is just a typo lol 

```suggestion
        have to split requests into <6 month intervals this can end up hitting that 
```